sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/retryprocessingerror/RetryProcessingErrorMLS.java

```diff
@@ -27,12 +28,27 @@ public class RetryProcessingErrorMLS<K, V> extends MessageListenerStrategy<K, V>
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RetryProcessingErrorMLS.class);
   private final MessageHandler<K, V> handler;
-  private final ErrorHandler<K, V> errorHandler;
+  private final ErrorHandler<K, V> intermediateErrorHandler;
+  private final ErrorHandler<K, V> finalErrorHandler;
+  private final long maxRetryCount;
   private String consumerName;
+  private Map<TopicPartition, OffsetCounter> offsetCounters =
+      Collections.synchronizedMap(new HashMap<>());
 
-  public RetryProcessingErrorMLS(MessageHandler<K, V> handler, ErrorHandler<K, V> errorHandler) {
+  public RetryProcessingErrorMLS(
+      MessageHandler<K, V> handler, ErrorHandler<K, V> intermediateErrorHandler) {
+    this(handler, intermediateErrorHandler, Long.MAX_VALUE, null);
```

Comment:
Probably very unlike to reach max long, but what about using `-1` as _forever_?

Do you know if there is any increasing delay for the next try? Afaik the default polling is quite quick, so we could reach high numbers (not max long) early - maybe before any subsequent failing service has time to recover.

Reply:
Found it: https://github.com/SDA-SE/sda-dropwizard-commons/blob/52395ce450c4674d1cf9dbba0bc3a4ebce5e7a37/sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/MessageListener.java#L180

Reply:
When using `-1` I need a special handling for that value. Using MAX_VALUE I can just use the default logic